### ISSUE TITLE
Setting customized port numbers in /etc/service doesn't work

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#985]
Originally reported of Google Code with ID 929.

```
<b>What steps will reproduce the problem?</b>
1. Add a entry like

      ganeti-noded	1899/tcp

   into /etc/services

2. Restart Ganeti

<b>What is the expected output? What do you see instead?</b>

The Haskell daemons aren't able to connect to noded, because they're not reading the services entry.

Also Haskell daemons don't honor the services entries for the ports they use (like confd).
```
Originally added on 2014-08-27 11:33:43 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2014-10-15 10:00:30 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit aff5a10227e483088d5a28efd04efc13ec771037
Author: Klaus Aehlig <aehlig@google.com>
Date:   Wed Oct 15 11:58:18 2014 +0200

    Look up RPC port
    
    ...instead of hard-codedly using the default port.
    Ganeti's noded honors /etc/services, so also look
    up the port there when trying to contact the daemon.
    
    Signed-off-by: Klaus Aehlig <aehlig@google.com>
    Reviewed-by: Helga Velroyen <helgav@google.com>

```
Originally added on 2014-10-15 13:36:39 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
This fixes the RPC issue; what is still remaining is the haskell daemons themselves
honoring /etc/services.
```
Originally added on 2014-10-15 13:38:09 +0000 UTC.

Added Labels: Priority-Low
Removed Labels: Priority-Medium Component-luxi

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit 4b41183d2f6483185fa85b633920e3731b68f0fc
Author: Klaus Aehlig <aehlig@google.com>
Date:   Tue Feb 10 14:51:29 2015 +0100

    Make ganeti-mond honor /etc/services
    
    If no port is specified as a command-line option, prefer
    the value of /etc/services over the global default.
    
    Signed-off-by: Klaus Aehlig <aehlig@google.com>
    Reviewed-by: Helga Velroyen <helgav@google.com>


This was the last daemon not honoring /etc/services. Hence marking as fixed.
```
Originally added on 2015-02-10 15:36:56 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.14.0
```
Originally added on 2015-06-02 10:37:02 +0000 UTC.

Changed State: Released
